src/generic_ui/polymer/cloud-install.ts

```diff
@@ -23,12 +20,16 @@ Polymer({
     this.$.failureDialog.close();
   },
   loginTapped: function() {
+    var install_args = {
+        providerName: DEFAULT_PROVIDER,
+        region: this.$.regionMenu.selected
+    };
     this.closeDialogs();
     // TODO: show the dialog when this value changes, not this nasty hack
     ui.cloudInstallStatus = '';
     this.$.installingDialog.open();
 
-    ui.cloudInstall(INSTALL_ARGS).then((result: uproxy_core_api.CloudInstallResult) => {
+    ui.cloudInstall(install_args).then((result: uproxy_core_api.CloudInstallResult) => {
```

Comment:
No need for a named variable here, you could just do:
```typescript
ui.cloudInstall({
  providerName: DEFAULT_PROVIDER,
  region: this.$.regionMenu.selected
}).then((result: uproxy_core_api.CloudInstallResult) => {
  // do stuff
});
```

(if you prefer keep the variable, make it `const` and give it a camel-cased name)